app/mailers/access_request_mailer.rb

```diff
@@ -1,10 +1,11 @@
 class AccessRequestMailer < ApplicationMailer
-  def access_request_email(host, user, manager_email, reason, project_ids)
+  def access_request_email(host, user, manager_email, reason, project_ids, role_id)
     @host = host
     @user = user
     @manager_email = manager_email
     @reason = reason
     @projects = Project.order(:name).find(project_ids)
+    @role = Role.find(role_id)
```

Comment:
Same as above. Actually, be careful, because in the context of a project, the available roles are:
- Deployer, with id: 0
- Admin: with id: 1